app/policies/dataservice/blob_policy.rb

```diff
@@ -0,0 +1,15 @@
+class Dataservice::BlobPolicy < ApplicationPolicy
+
+  def index?
+    admin?
+  end
+
+  def new_or_create?
+    admin?
```

Comment:
The old controller had this code which only allowed admin access:

    before_filter :admin_only, :except => [:show]

    def admin_only 
      unless is_admin?
        login_redirect
      end
    end